config/locales/fra/views/backend/parcels.yml

```diff
@@ -0,0 +1,5 @@
+fra:
+  backend:
+    parcels:
+      outgoing_item_fields:
+        product_out_of_stock: Le produit n'est pas en stock
```

Comment:
Please use standard use of translations, so use `.tl` method for now and remove this file. And overall, English translation must be defined first before others.